examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>This notebook proposes research on how different learning rate schedulers affect deep learning model performance on image classification task. The dataset in use is Imagenette160 and the model is ResNet34. Research process itself is done via running several repetitions of model training with specific learning rate scheduler from predefined domain of options. Model evaluation is done periodically during each of these runs on test part of the dataset. Plots and aggregated metrics at the end of the notebook clearly show the difference between experiments with different learning rate schedulers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>